src/script/engine.zig

```diff
@@ -455,6 +473,22 @@ test "Script execution - OP_RETURN" {
     }
 }
 
+test "Script execution - OP_TOALTSTACK OP_FROMALTSTACK" {
+    const allocator = std.testing.allocator;
+
+    // Simple script: OP_1 OP_1 OP_EQUAL
+    const script_bytes = [_]u8{ 0x51, 0x6b, 0x6c };
```

Comment:
when https://github.com/zig-bitcoin/btczee/pull/47 is merged please use it instead of the raw bytes value. Much more readable. 
I will ping you when it's merged